Makefile

```diff
@@ -16,19 +16,19 @@ CRD_OPTIONS ?= "crd:crdVersions=v1,trivialVersions=false"
 all: test varnish-operator varnish-controller
 
 # Run tests
-test: generate fmt vet manifests
```

Comment:
What is the reason for removing `fmt` from all of these commands?

Reply:
This is something we need to discuss actually. the makefile's fmt target is commented out for now. It was calling `goimports` which kept removing `github.com/caarlos0/env/v6` for some reason.

Reply:
This is the error:
```
> make test
/Users/cin/go/bin/controller-gen object:headerFile=./hack/boilerplate.go.txt paths="./..."
cd /Users/cin/go/src/varnish-operator/ && goimports -w ./pkg ./cmd ./api
cd /Users/cin/go/src/varnish-operator/ && go vet ./pkg/... ./cmd/... ./api/...
# github.com/ibm/varnish-operator/pkg/varnishcluster/config
pkg/varnishcluster/config/config.go:47:34: undefined: env
pkg/varnishcluster/config/config.go:56:12: undefined: env
# github.com/ibm/varnish-operator/pkg/varnishcontroller/config
pkg/varnishcontroller/config/config.go:40:16: undefined: env
pkg/varnishcontroller/config/config.go:49:23: undefined: env
pkg/varnishcontroller/config/config.go:55:30: undefined: env
pkg/varnishcontroller/config/config.go:61:11: undefined: env
make: *** [vet] Error 2
```

Reply:
old goimports version bit me here...